bevy/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "bevy_mongodb_asset"
+version = "0.1.0"
+authors = ["Abraham Egnor <abraham.egnor@mongodb.com>"]
+license = "Apache-2.0"
+repository = "https://github.com/mongodb/mongo-rust-driver"
+edition = "2024"
+
+[workspace]
```

Comment:
This declares that the crate is _not_ part of the shared workspace defined in the parent directory, mostly because bevy pulls in a ton of dependencies and it didn't seem worth it to dump them all in to that `Cargo.lock`.